width.go

```diff
@@ -1,5 +1,7 @@
 package uniseg
 
+var EastAsianAmbiguousWidth = 1
```

Comment:
Please add the following comment here:

> EastAsianAmbiguousWidth specifies the monospace width for East Asian characters classified as Ambiguous. The default is 1 but some rare fonts render them with a width of 2.

Make sure to maintain the 80 character line width.